invenio_search/ext.py

```diff
@@ -138,48 +138,46 @@ def _walk_dir(aliases, *parts):
         # Start the recursion here:
         _walk_dir(self.aliases, alias)
 
-    def register_templates(self, directory):
-        """Register templates from the provided directory.
+    def register_templates(self, module):
+        """Register templates from the provided module.
 
-        :param directory: The templates directory.
+        :param module: The templates module.
         """
         try:
-            resource_listdir(directory, 'v{}'.format(ES_VERSION[0]))
-            directory = '{}/v{}'.format(directory, ES_VERSION[0])
+            resource_listdir(module, 'v{}'.format(ES_VERSION[0]))
+            module = '{}.v{}'.format(module, ES_VERSION[0])
```

Comment:
I am not sure about the naming in this case since by adding the version in the path you are not referring to a module anymore.